### PR TITLE
[RU] Translation of the last five news

### PR DESCRIPTION
**Pull request by lex111**
I'm starting to translate the news! :smiley:

### CODE REVIEW

File: ru/news/_posts/2019-03-13-ruby-2-5-4-released.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+author: "nagachika"
+translator: "lex111"
+date: 2019-03-13 11:30:00 +0000
+lang: ru
+---
+
+Вышел релиз Ruby 2.5.4.
+
+Этот релиз включает исправления ошибок и обновление безопасности RubyGems.
+Подробную информацию смотрите в [Многочисленные уязвимости в RubyGems](/en/news/2019/03/05/multiple-vulnerabilities-in-rubygems/)
```
Comment (razum2um) [inline ru/news/_posts/2019-03-13-ruby-2-5-4-released.md]: @lex111 maybe it makes sense to translate this one as well, just to have proper consistent links from scratch?

File: ru/news/_posts/2019-03-31-support-of-ruby-2-3-has-ended.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+date: 2019-03-31 00:00:00 +0000
+lang: ru
+---
+
+Мы объявляем, что вся поддержка версий Ruby 2.3 закончилась.
+
+28 марта 2018 года после выхода Ruby 2.3.7 поддержка версий Ruby 2.3 перешла на стадию поддержки безопасности.
+Теперь, спустя год, этот этап завершился.
+Поэтому 31 марта 2019 года вся поддержка релизов Ruby 2.3 заканчивается.
+Исправления ошибок и изменения, касающиеся безопасности из новых версий Ruby больше не будут перенесены в ветку 2.3.
```
Comment (razum2um) [inline ru/news/_posts/2019-03-31-support-of-ruby-2-3-has-ended.md]:
```suggestion
Исправления ошибок и изменения, касающиеся безопасности, из новых версий Ruby больше не будут перенесены в ветку 2.3.
```

File: ru/news/_posts/2019-03-31-support-of-ruby-2-3-has-ended.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+Там также не будет никаких патчей из 2.3.
+Мы крайне рекомендуем вам как можно скорее обновиться до Ruby 2.6 или 2.5.
+
+## О поддерживаемых в данный момент версиях Ruby
+
+### Версии Ruby 2.6
+
+Сейчас в обычном режиме поддержки.
+Мы будем применять исправлениях ошибок из последний версий и выпускать новые релизы по мере необходимости.
```
Comment (razum2um) [inline ru/news/_posts/2019-03-31-support-of-ruby-2-3-has-ended.md]:
```suggestion
Мы будем применять исправлениях ошибок из последних версий и выпускать новые релизы по мере необходимости.
```

File: ru/news/_posts/2019-03-31-support-of-ruby-2-3-has-ended.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+### Версии Ruby 2.5
+
+Сейчас в обычном режиме поддержки.
+Мы будем применять исправлениях ошибок из последний версий и выпускать новые релизы по мере необходимости.
+А если будет обнаружена критическая проблема, связанная с безопасностью, мы выпустим версию с ее исправлением.
+
+### Версии Ruby 2.4
+
+Сейчас только поддерживаются обновления, связанные с безопасностью.
```
Comment (razum2um) [inline ru/news/_posts/2019-03-31-support-of-ruby-2-3-has-ended.md]:
```suggestion
Сейчас поддерживаются только обновления, связанные с безопасностью.
```

File: ru/news/_posts/2019-03-31-support-of-ruby-2-3-has-ended.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+Мы не будем переносить какие-либо исправления ошибок, сделанные в новые версии, если они не связаны с безопасностью.
```
Comment (razum2um) [inline ru/news/_posts/2019-03-31-support-of-ruby-2-3-has-ended.md]:
```suggestion
Мы не будем переносить какие-либо исправления ошибок, сделанные в новых версиях, если они не связаны с безопасностью.
```

File: ru/news/_posts/2019-03-13-ruby-2-5-4-released.md (hunk 1 above)
Comment (lex111) [reply ru/news/_posts/2019-03-13-ruby-2-5-4-released.md]: @razum2um thank you for the review!

>  maybe it makes sense to translate this one as well, just to have proper consistent links from scratch?

No, I don't think so.

**Pull request merged**